src/services/programService/index.js

```diff
@@ -330,7 +333,12 @@ export const listDataCenters = async (shortName, jwt) => {
 		.then((response) => response.json())
 		.then((data) => {
 			if (data && Array.isArray(data)) {
```

Comment:
this can just be `Array.isArray(data)` , the isArray method handles undefined vs something like `Array.isArray(data.x)` where `x` will be looked up first and may not exist